charts/eks-operator/values.yaml

```diff
@@ -9,3 +9,4 @@ eksOperator:
 httpProxy: ""
 httpsProxy: ""
 noProxy: ""
+additionalTrustedCAs: false
```

Comment:
Should we document this in app-readme or readme of chart ? cc @rmweir

Reply:
Oh nvm, forgot how this worked.

Reply:
@cbron what do you mean? I think putting it in a readme is a good idea.